composer/algorithms/low_precision_layernorm/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "low_precision_layernorm": {
+        "name": "Low Precision LayerNorm",
+        "class_name": "LowPrecisionLayerNorm",
+        "functional": "apply_low_precision_layernorm",
+        "tldr": "Forces LayerNorm to fp16 or bf16",
+        "attribution": "",
+        "link": "",
+        "domains": [
+            "nlp"
```

Comment:
I'm not sure I'd call LayerNorm an NLP op. I think it's more general.

Reply:
Added CV tag as well.